components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.ts

```diff
@@ -169,14 +192,14 @@ export class PolicyAddMembersComponent implements OnInit, OnDestroy {
     this.store.select(addPolicyMembersStatus).pipe(
       filter(identity),
       takeUntil(pendingAdd))
-      .subscribe((state) => {
-        if (state === EntityStatus.loadingSuccess) {
+      .subscribe((addPolicyState) => {
```

Comment:
variable `state` was only changed because it became a shadow now that we are importing `state` from the angular animations API